.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+**Which issue is resolved by this Pull Request:** 
+Resolves #
+
+**Description of your changes:**
+
+**Environment tested:**
+
+* Tekton Version (use `tkn version`):
+* Kubernetes Version (use `kubectl version`):
+* OS (e.g. from `/etc/os-release`):
```

Comment:
Should we include the Python version here as well, just to be complete?

Reply:
done